doc/DeveloperGuide.md

```diff
@@ -88,6 +88,16 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use Case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of tags
+3. User requests to rename a specific tag in the list
+4. User enters new name for the tag
+5. AddressBook confirms the change with the user before renaming the tag
+
```

Comment:
- What happens when the new tag is already present in the address book?
- What happens if the new tag is same as the old tag?